cirq-core/cirq/interop/quirk/url_to_circuit_test.py

```diff
@@ -371,21 +371,21 @@ def test_completes_weight_zero_billion_laughs():
 
 def test_example_qft_circuit():
     qft_example_diagram = """
```

Comment:
We should avoid making these kinds of diagrammatic changes, precisely because people use them in tests that need to be updated when we do so.

Reply:
I agree it's a breaking change, but I think it's unavoidable here since we are changing the type returned by `controlled_by`.